test/anthropic/internal/util_test.rb

```diff
@@ -228,7 +228,7 @@ def test_file_encode
       encoded = Anthropic::Internal::Util.encode_content(headers, body)
       cgi = FakeCGI.new(*encoded)
       assert_pattern do
-        cgi[""].read => ^val
```

Comment:
this is actually introduces a bug haha, `^val` means "assert `val` === `value`", while `=>  val` just assigns `cgi[""].read` to `val`

Reply:
Ugh, what a tricky syntax. Just pushed a commit to restore the pinned variables 👍